W-17700088-outbound-policies-fa (#757)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/policies-outbound-directory.adoc

= Outbound Policies Directory
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images

== Outbound Policies Included in Flex Gateway

[%header%autowidth.spread]
|===
|Policy |Category |Summary
| xref:policies-outbound-aws-lambda.adoc[AWS lambda] | Transformation | Triggers an AWS Lambda function from a standard HTTP request

> | xref:policies-outbound-circuit-breaker.adoc[Circuit Breaker] | Quality of Service | Prevents an upstream from receiving too many requests or connection

Reviewer: ```suggestion
| xref:policies-outbound-circuit-breaker.adoc[Circuit Breaker] | Quality of Service | Prevents an upstream from receiving too many requests or connections
```

| xref:policies-outbound-basic-auth.adoc[Credential Injection Basic Authentication] | Security | Allows access based on the basic authorization mechanism, with a single user-password
| xref:policies-outbound-oauth.adoc[Credential Injection OAuth 2.0] | Security | Adds headers to a request or a response

> | xref:policies-outbound-upstream-idle-timeout.adoc[Upstream Idle Timeout] | Quality of Service | Sets a maximum duration a stream between Flex Gateway and a upstream service can remain idle

Reviewer: ```suggestion
| xref:policies-outbound-upstream-idle-timeout.adoc[Upstream Idle Timeout] | Quality of Service | Sets the maximum idle duration for a stream between Flex Gateway and an upstream service
```

= Outbound Policies Directory
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images

== Inbound Policies Included in Flex Gateway

[%header%autowidth.spread]
|===
|Policy |Category |Summary
| xref:policies-outbound-basic-auth.adoc[Basic Authentication: Simple] | Security | Allows access based on the basic authorization mechanism, with a single user-password

> | xref:policies-outbound-message-logging.adoc[Message Logging] | Troubleshooting | Logs custom messages using information from outgoing requests, responses sent to the client, or information from other policies applied to the same API endpoint

Reviewer 2: this should not be included in 1.9
